docs/infrastructure_and_maintenance/performance.md

```diff
@@ -52,6 +52,17 @@ In production setups:
 - Always enable opcache for php-fpm/`mod_php`.
 - Prefer php-fpm and web server using it over fast-cgi for lower overall memory usage.
 
+!!! caution
+
+    If you are running [[= product_name =]] in version v4.6.8 and below, to avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, run the following commands:
```

Comment:
```suggestion
    To avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, v4.6.x needs an additional error handler. The error handler was added by default for projects that started with 4.6.8 or later. For projects that started on earlier versions, it needs do be added manually by running the command:
```
^^ sounds not very good language wise - main thing is that you might be on 4.6.8 or later now but just missed to add the error handler.